### PR TITLE
dpg, cadl, upload to cadl ranch report

### PR DESCRIPTION
**Pull request by haolingdong-msft**
ref: https://github.com/Azure/cadl-ranch/blob/main/docs/using-cadl-ranch.md#upload-coverage

### CODE REVIEW

**Review by weidongxu-microsoft — APPROVED** [review]
to unblock you

File: eng/pipelines/cadl-ci.yaml [hunk 1]
```diff
@@ -135,3 +135,11 @@ jobs:
         artifact: CoverageReport
         displayName: 'Publish Cadl Ranch Coverage Report'
 
+      - task: AzureCLI@2
+        displayName: 'Upload to Cadl Ranch Coverage Report'
+        inputs:
+          azureSubscription: "Azure SDK Playground"
+          scriptType: "bash"
+          scriptLocation: "inlineScript"
+          inlineScript: cadl-ranch upload-coverage --coverageFile ./cadl-ranch-coverage-java.json --generatorName java --storageAccountName cadlranchcoverage --generatorVersion 0.1.0
```
Comment (timotheeguerin) [inline eng/pipelines/cadl-ci.yaml:144]: `--generatorVersion 0.1.0`

just a note to make sure to retrieve the real version for this

Comment (haolingdong-msft) [reply eng/pipelines/cadl-ci.yaml:144]: Sure, I will update

**Pull request merged**